components/cam/src/physics/cam/clubb_intr.F90

```diff
@@ -2500,36 +2500,37 @@ real(r8) :: ugust  ! function: gustiness as a function of convective rainfall
 
 !PMA adds gustiness and tpert
 
-    vmag_gust(:)    = 1._r8
+    vmag_gust(:)    = 0._r8
```

Comment:
@polunma :This is one of the changes causing NBFB behavior. Reverting it back to 1._r8 fixes it.